templates/blazor-base/csharp/tab/GettingStarted.txt

```diff
@@ -0,0 +1,40 @@
+Welcome to the Teams Toolkit! Here are some tips to get started building your app.
```

Comment:
Looks like this GettingStarted.txt is the same as Bot one. Can we share them instead of duplicating them?